roles/etcd/tasks/system_container.yml

```diff
@@ -15,6 +15,57 @@
       {%- endif -%}
       {% endfor -%}
 
+- name: Check etcd system container package
+  command: >
+    atomic containers list --no-trunc -a -f container=etcd -f backend=ostree
+  register: etcd_result
+
+- name: Unmask etcd service
+  systemd:
+    name: etcd
+    state: stopped
+    enabled: yes
+    masked: no
+    daemon_reload: yes
+  register: task_result
+  failed_when: task_result|failed and 'could not' not in task_result.msg|lower
```

Comment:
Does the `failed_when` works with:
```yaml
failed_when:
- task_result|failed
- 'could not' not in task_result.msg|lower
```
as well?

Plus, should not the second condition be `'could not' in task_result.msg|lower`? If the condition is ok, can be add comment why is that so?